pyproject.toml

```diff
@@ -0,0 +1,145 @@
+[build-system]
+requires = ['hatchling']
+build-backend = 'hatchling.build'
+
+[tool.hatch.version]
+path = 'pydantic/version.py'
+
+[project]
+name = 'pydantic'
+description = 'Data validation using Python type hints'
+authors = [
+    {name = 'Samuel Colvin', email = 's@muelcolvin.com'},
+    {name = 'Eric Jolibois', email = 'em.jolibois@gmail.com'},
+    {name = 'Hasan Ramezani', email = 'hasan.r67@gmail.com'},
```

Comment:
@PrettyWood @hramezani I assume you're happy with this?

Let me know if not. I took your email addresses from github profiles, hope that's okay.

Reply:
Good for me! Thank you:)

Reply:
Fine for me! Thank you.

Reply:
thanks both.